api/docs/v1.52.yaml

```diff
@@ -5115,6 +5151,7 @@ definitions:
       HostConfig:
         $ref: "#/definitions/HostConfig"
       GraphDriver:
+        x-nullable: true
```

Comment:
This line is missing in the main `swagger.yaml` (it's only in the `v1.52.yaml` 😅

Reply:
Wow nice catch and also I forgot to add the Storage field in the container inspect response. Added in https://github.com/moby/moby/compare/e1d4ad209d70f3b34c49d177e9a6e182a6206905..efa077848f912b1c8febe579f345387e38008eb0